src/hidimstat/conditional_feature_importance.py

```diff
@@ -1,8 +1,7 @@
 import numpy as np
 from joblib import Parallel, delayed
-from sklearn.base import check_is_fitted, clone, BaseEstimator
+from sklearn.base import BaseEstimator, check_is_fitted, clone
 from sklearn.metrics import root_mean_squared_error
-from sklearn.utils.validation import check_random_state
```

Comment:
Don't remove this line.
It's important to keep the check on the random generator from scikit learn.